incubator/tensorflow-serving/templates/pvc.yaml

```diff
@@ -0,0 +1,20 @@
+{{- if .Values.persistence.enabled }}
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{ .Release.Name }}-pvc
```

Comment:
Add standard labels. Name should be `{{ template "tensorflow-serving.fullname" . }}`.